main.sh

```diff
@@ -166,21 +166,21 @@ ebpf() {
 		mkdir -p /tmp/elfutils-source
 		cd /tmp/elfutils-source
 		yumdownloader --source elfutils
-		rpm2cpio elfutils-0.189-3.el9.src.rpm | cpio -iv
+		rpm2cpio elfutils-0.190-2.el9.src.rpm | cpio -iv
```

Comment:
@rootfs , @maryamtahhan to keep consistent with https://github.com/sustainable-computing-io/kepler/blob/main/build/Dockerfile.builder#L11 
please considering rollback the change for libbpf or bump up libbpf version in kepler build image.
either fine for me.

Reply:
Absolutely agree with @SamYuan1990 , lets go in lock step with kepler dependencies.

Reply:
I update these as they were causing the unit tests to fail - so i imagine the Kepler build image needs to be  updated. I will push a PR to kepler

Reply:
created PR https://github.com/sustainable-computing-io/kepler/pull/1408